doc/devel/gitwash/development_workflow.rst

```diff
@@ -397,7 +397,7 @@ Deleting a branch on GitHub_
    # delete branch on GitHub
    git push origin :my-unwanted-branch
 
-(Note the colon ``:`` before ``test-branch``.  See also:
+Note the colon ``:`` before ``my-unwanted-branch``.  See also:
 http://github.com/guides/remove-a-remote-branch
```

Comment:
No, that link is for deleting a remote itself, not the a remote branch. I think the correct link is https://help.github.com/articles/pushing-to-a-remote/#deleting-a-remote-branch-or-tag